canvas_sdk/commands/base.py

```diff
@@ -66,23 +70,25 @@ def command_schema(cls) -> dict:
 
     def originate(self) -> Effect:
         """Originate a new command in the note body."""
-        if not self.note_id:
+        if not self.note_uuid:
             raise AttributeError("Note id is required to originate a command")
-        return {
-            "type": f"ADD_{self.Meta.key.upper()}_COMMAND",
-            "payload": {
-                "user": self.user_id,
-                "note": self.note_id,
-                "data": self.values,
-            },
-        }
+        return Effect(
+            type=EffectType.Value(f"ORIGINATE_{self.constantized_key()}_COMMAND"),
+            payload=json.dumps(
+                {
+                    "user": self.user_id,
+                    "note": self.note_uuid,
+                    "data": self.values,
+                }
+            ),
+        )
 
     def edit(self) -> Effect:
         """Edit the command."""
         if not self.command_uuid:
             raise AttributeError("Command uuid is required to edit a command")
         return {
-            "type": f"EDIT_{self.Meta.key.upper()}_COMMAND",
+            "type": f"EDIT_{self.constantized_key()}_COMMAND",
```

Comment:
I changed these from `self.Meta.key.upper()` to a call to the new method `constantized_key` to account for multi-word command types. Instead of `EDIT_MEDICATIONSTATEMENT_COMMAND` we want the more readable `EDIT_MEDICATION_STATEMENT_COMMAND`.